pkg/pillar/build.yml

```diff
@@ -16,10 +16,13 @@ config:
     - /persist:/persist:rshared,rbind
     - /usr/bin/containerd:/usr/bin/containerd
     - /opt/vendor:/opt/vendor:ro
+    - /sys/fs/cgroup:/sys/fs/cgroup:rshared,rbind,rw
+
   net: host
   capabilities:
     - all
   pid: host
+  cgroupns: host
```

Comment:
As discussed offline, I don't think this is a valid setting here.

Reply:
As we discussed offline, I think that there is no option for this currently in lkt pkg. It should belong in `config`, but there is nothing there. If this works, then that is great; if you need something added, then open an issue in lkt and I will take care of it.